packages/cactus-plugin-keychain-vault/openapitools.json

```diff
@@ -2,6 +2,6 @@
   "$schema": "node_modules/@openapitools/openapi-generator-cli/config.schema.json",
   "spaces": 2,
   "generator-cli": {
-    "version": "5.2.0"
+    "version": "6.0.1"
```

Comment:
@jagpreetsinghsasan Is this version devation a must for fixing the issue? I tried to change it to 5.2.1 just now and the container build worked regardless (e.g. your Rust code fixes seem to work anyway with my smoke test but I did not run the test cases though).

If it's needed, then fine, but otherwise I'd leave this change out so that it doesn't conflict with your other PR upgrading all the OpenAPI generator versions in lockstep.
(Also now that I'm seeing that the generator has a 6.x release, another PR could be sent to upgrade to that as long as we have no issues with it)

`packages/cactus-plugin-keychain-vault/openapitools.json`

Reply:
@petermetz I have tested it from my end as well, it indeed works with `5.2.1` version. I am about to update the same and push it to the PR branch asap
![image](https://user-images.githubusercontent.com/56861721/199165898-f339b5d1-5b90-4070-9fd1-ff287dcb2ad6.png)